README.md

```diff
@@ -18,21 +18,32 @@ It will allow you, whether you are developing on macOS, Windows or Linux to quic
 * Install [VScode](https://code.visualstudio.com/download)
 * Open a terminal and clone `app-boilerplate` with `git clone git@github.com:LedgerHQ/app-boilerplate.git`.
 * Open the `app-boilerplate` folder with VSCode.
+* Install the recommended extensions
 * Open the vscode tasks with  `ctrl + shift + b` (`command + shift + b` on a Mac) and run the following actions :
-    * Pull and run the [ledger-app-dev-tools](https://github.com/LedgerHQ/ledger-app-builder/pkgs/container/ledger-app-builder%2Fledger-app-dev-tools) docker image by selecting `Run dev-tools image` (select the task for your platform).
-    * Build the for the device model of your choice with `Build app`.
-    * Test your binary on [Speculos](https://github.com/LedgerHQ/speculos) with `Run Speculos`.
+    * Pull and run the [ledger-app-dev-tools](https://github.com/LedgerHQ/ledger-app-builder/pkgs/container/ledger-app-builder%2Fledger-app-dev-tools) docker image by selecting `[setup] Run dev-tools image`.
+    * Select a target (either when editing a C file, from the bottom right of the status bar, or with `command + shift + p` and `C/C++: Select a configuration...`)
```

Comment:
need to have also the command for Windows & Mac

Reply:
Did an update of the doc